docs/content/guide/getting-started.md

```diff
@@ -1,40 +1,51 @@
 # Getting Started
 
-## Installing Spica
+## Introduction
 
-This tutorial helps you to install Spica to different environments. We provide a few ways to install, so you can choose which one of them suits you. 
+These documents will take you from 0 to hero and helps you to discover advanced features of Spica Development Engine.
 
-### Using Spica Cli
+### What is Spica?
 
-> Note: This feature is currently unavailable. Continue with other installation ways. 
+Spica (a.k.a. Spica Development Engine) is an open-source package that gives virtually everything a backend developer needs. It gives a pre-built administration panel and a fully manageable no-code REST API. It can be used as a backend service, a database layer, or as a headless CMS for managing content.
+
+### Differences Between Spica API and Spica Client
+
+Spica Development Engine divided into two parts because of principal differences.
+
+Spica Client, is a user-friendly admin panel for those who like to manage their development process from a panel with a few clicks.
+
+Spica API, on the other hand, is a fully controllable REST API.
+
+As Spica has an API-first approach, any features of the engine are controllable via HTTP calls. Also, it’s suitable to integrate with any frontend application to free the product owners from backend development costs.
+
+## Installation
+
+This tutorial helps you to install Spica to different environments. We provide a few ways to install, so you can choose which one of them suits you.
+
+### Using Spica CLI
 
 Spica has a command-line interface for quick installation. To use it, you must have [Docker](https://www.docker.com/) and [NodeJs](https://nodejs.org) installed on your development environment.
 
+> IMPORTANT: There has to be at least one docker machine is up and running on your environment. To create a docker machine enter the following command to your terminal:
+>
+> ```shell
+> $ docker-machine create <machine name>
+> $ eval "$(docker-machine env <machine name>)"
+> ```
+
```

Comment:
On our test with @htuna07, we couldn't run a Spica instance locally without using those commands. We were using npm docker only. Maybe with docker desktop, it may be unnecessary.